### ISSUE TITLE
Color contrast between text and background doesn't meet WCAG AA standards

### ISSUE THREAD

**Issue by metalpail** [StarCitizenTools/mediawiki-skins-Citizen#1926]
### Description

The color contrast between text and background fails to meet WCAG AA standards for the Create/Edit buttons in dark mode and certain syntax highlighting in code blocks in light mode.

<img width="188" height="115" alt="Image" src="https://github.com/user-attachments/assets/c840ccb4-01e8-412d-bd06-dba1666b7b79" />

<img width="252" height="90" alt="Image" src="https://github.com/user-attachments/assets/003ecc66-0477-4aa1-a7fd-46e9b264818e" />

### Steps to reproduce

Go to any article or module page

### Expected behavior

The color contrast ratio between the text and background for these elements should be at least 4.5:1 to meet WCAG AA standards.

### Citizen version

Latest release

### MediaWiki version

1.45

**Comment by alistair3149** [comment]
They are addressed in the new tokens as a part of [Citizen v4](https://mwcitizen.skin/guide/migrating-to-citizen-4).

**Comment by alistair3149** [comment]
I misread the issue. The code syntax color wasn't in the v4 pass, I will do an audit.
